build/agones-bot/cloudbuild.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+steps:
+  - name: gcr.io/cloud-builders/docker
+    args:
+      - build
+      - --tag=${_REGISTRY}/agones-bot:latest
+      - '.'
+images:
+  - ${_REGISTRY}/agones-bot
+options:
+  dynamic_substitutions: true
+substitutions:
+  _REGISTRY: "us-docker.pkg.dev/${PROJECT_ID}/ci"
```

Comment:
The CI registry has a cleanup policy of every 40 days. 

So two options:

1. Put it somewhere else.
2. Setup a cron trigger that rebuilds the images a week (?) so it's always refreshed.

Reply:
Made a `build/` repo.